classes/classes/Creature.as

```diff
@@ -140,7 +140,16 @@ package classes
 				
 		//Appearance Variables
 		//Gender 1M, 2F, 3H
-		public var gender:int = GENDER_NONE;
+		public function get gender():int
```

Comment:
Tests that cover this function would be nice, as they will let you know if the behaviour of the function has changed.
This would be useful for future refactoring efforts.